components/contribution-flow/ContributionBlocker.js

```diff
@@ -85,6 +90,36 @@ export const getContributionBlocker = (loggedInUser, account, tier, shouldHaveTi
     return { reason: CONTRIBUTION_BLOCKER.TIER_EXPIRED, type: 'warning', showOtherWaysToContribute: true };
   } else if (account.settings.disableCustomContributions && !tier) {
     return { reason: CONTRIBUTION_BLOCKER.NO_CUSTOM_CONTRIBUTION, type: 'warning', showOtherWaysToContribute: true };
+  } else if (
+    tier &&
+    tier.interval !== null &&
+    !account.host.supportedPaymentMethods.some(paymentMethod => paymentMethods.includes(paymentMethod))
+  ) {
+    return {
+      reason: CONTRIBUTION_BLOCKER.NO_PAYMENT_PROVIDER,
+      type: 'warning',
+      showOtherWaysToContribute: true,
+      content: (
+        <React.Fragment>
+          <strong>
+            <FormattedMessage
+              id="ContributionFlow.noStripeOrPaypalConfigured"
+              defaultMessage="No payment provider available for accepting recurring contributions. Your host admin need to configure Stripe or PayPal."
```

Comment:
This is not a suitable message to display to users: contributors will see that too, it's not only for host admins. I would suggest using the same message as the previous check (`!account.host.supportedPaymentMethods?.length`). It could be nice to abstract all that in something like `hasPaymentMethodsAvailable` to simplify the if/else if.

Reply:
Done. 👍🏽